python/jtmodules/src/jtmodules/expand_or_shrink.py

```diff
@@ -37,11 +42,27 @@ def main(image, n, plot=False):
     jtmodules.expand_objects.Output
     '''
     # NOTE: code from CellProfiler module "expandorshrink"
-    background = image == 0
-    distance, (i, j) = ndi.distance_transform_edt(background, return_indices=True)
+    # NOTE (S.B. 25.1.2018): renamed from "expand" to "expand_or_shrink"
     expanded_image = image.copy()
-    mask = background & (distance < n)
-    expanded_image[mask] = image[i[mask], j[mask]]
+    print n
```

Comment:
Looks like a debugging statement that slipped in?

Reply:
Yes, please delete